From: Hrvoje Senjan <hrvoje.senjan@gmail.com>
Date: Tue, 20 Aug 2013 11:35:57 +0000
Subject: Partially revert commit 83d3018
X-Git-Url: http://quickgit.kde.org/?p=kde-workspace.git&a=commitdiff&h=bb2bddb083eb293786206ac91ae1feeaf942db32
---
Partially revert commit 83d3018

Somehow unique made ksplash quit too soon and/or plasma start
too late

REVIEW: 112073
BUG: 321695
FIXED-IN: 4.11.1
---


--- a/plasma/desktop/shell/data/plasma-desktop.desktop
+++ b/plasma/desktop/shell/data/plasma-desktop.desktop
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Exec=plasma-desktop
-X-DBUS-StartupType=unique
+X-DBUS-StartupType=multi
 Name=Plasma Desktop Workspace
 Name[ar]=مساحة عمل بلازما
 Name[ast]=Espaciu de trabayu del escritoriu Plasma

